images/node/scripts/origin-node-run.sh

```diff
@@ -17,6 +17,6 @@ if [ ! -f ${HOST_ETC}/systemd/system/docker.service.d/docker-sdn-ovs.conf ]; the
     cp /usr/lib/systemd/system/docker.service.d/docker-sdn-ovs.conf ${HOST_ETC}/systemd/system/docker.service.d
 fi
 
-/usr/bin/openshift start node "--config=${conf}" "${opts}"
+/usr/bin/openshift start node "--config=${conf}" "${opts}" &
```

Comment:
@Miciah I attempted to get rid of the backgrounding and signal trapping under the assumption that when docker sends a SIGTERM to this container it would send that on to openshift however that's not happening. Any idea why?